scripts/globals/abilities/benediction.lua

```diff
@@ -36,6 +36,11 @@ ability_object.onUseAbility = function(player, target, ability)
         heal = maxHeal
     end
 
+    local power = 33 --chance to remove Doom. Basing off of Holy Water?
+
+    if (target:hasStatusEffect(xi.effect.DOOM) and power > math.random(1, 100)) then
```

Comment:
Style guide: Outer parenthesis ( ) arent needed.

Also, IDK why fixing benediction is added here. Not a bad thing, but weird you didn't comment it.

Reply:
I have no idea either... I pulled a new repository, made the adjustments and pushed to my fork. IDK maybe I should watch the instructional video for this

Reply:
I suggest basing it on whatever Cursna does. Which might be the same as Holy Water, I didn't look. :stuck_out_tongue: